zawata/S19-Team2
# Adding in containerization of project

This PR brings in changes to fully build our project via docker, allowing us to run the full project pipeline (C -> Flask -> Web Page) without having to worry about specific OS issues. 

To Test:

- [ ] Download and Install [Docker](https://www.docker.com/get-started)
Once you have docker installed, _make sure_ that you have the _docker daemon running_

- [ ] **Build the project**
By running the command (inside the project directory): `docker build -t space .`
(This should take a long time the first time you run it)

- [ ] **Run the containerized project**
Run the following command: `docker run -p 5000:5000 space`

- [ ] **Make sure that the project runs**
You should see the webpage at http://localhost:5000

Now whenever you make a code change, you can run the above two commands to test the full pipeline. If only working on frontend code that doesn't rely on the backend, you can still use the same `npm run dev` command.
